src/app/project/view-projects-logic/view-projects-logic.component.scss

```diff
@@ -127,7 +127,18 @@
 }
 
 .formatButtons {
-  margin: 5px;
+ // margin: 5px;
+  margin: 10px;
+}
+
+//added by Serge
```

Comment:
@sarmand19 Use an @Author decorator to mark your work.  No need to do this at the rule level though. Put it at the top of of the file for CSS.